content/foundations/color.mdx

```diff
@@ -2,77 +2,246 @@
+| ![](https://swatch-sid.vercel.app?mode=light&token=success.fg) `fg.success`     | Use to emphasize a positive message.                                |
+| ![](https://swatch-sid.vercel.app?mode=light&token=attention.fg) `fg.attention` | Use to highlight text or icons that require the user's attention.   |
+| ![](https://swatch-sid.vercel.app?mode=light&token=danger.fg) `fg.danger`       | Use to emphasize an error or a blocking status. Action is required. |
+| ![](https://swatch-sid.vercel.app?mode=light&token=severe.fg) `fg.severe`       | Use to emphasize a level of severity between attention and danger.  |
+| ![](https://swatch-sid.vercel.app?mode=light&token=done.fg) `fg.done`           | Use to style text that refers to completed tasks or workflows.      |
+
+### Backgrounds
+
+Background colors apply to surfaces of components or UI elements, such as pages, boxes, and overlays.
+
+| Foundations     | Usage                                                                                                              |
+| :-------------- | :----------------------------------------------------------------------------------------------------------------- |
+| ![](https://swatch-sid.vercel.app?mode=light&token=canvas.default) `bg.default`    | Primary background color.                                                                                          |
+| ![](https://swatch-sid.vercel.app?mode=light&token=canvas.subtle) `bg.subtle`     | Provides visual rest and contrast against the default background.                                                  |
+| ![](https://swatch-sid.vercel.app?mode=light&token=neutral.emphasisPlus) `bg.onEmphasis` | Provides the highest contrast against the default background, such as in tooltips.                                 |
+| ![](https://swatch-sid.vercel.app?mode=light&token=canvas.inset) `bg.inset`      | Can be used instead of the default background to create a focal point, such as in conversations or activity feeds. |
+
+| Color roles                                                    | Usage                                                                          |
+| :------------------------------------------------------------- | :----------------------------------------------------------------------------- |
+| ![](https://swatch-sid.vercel.app?mode=light&token=accent.subtle) `bg.accent`| Use to accentuate interactive areas in the UI like selected elements.          |
+| ![](https://swatch-sid.vercel.app?mode=light&token=success.subtle) `bg.success`                                                   | Use to highlight a positive message.                                           |
+| ![](https://swatch-sid.vercel.app?mode=light&token=attention.subtle)`bg.attention`                                                 | Use to highlight elements that require a user's attention or pending statuses. |
```

Comment:
Inconsistent space between swatch and text? would be nice to have a tiny bit of space between them

![image](https://user-images.githubusercontent.com/1863771/146937114-cad68604-73b6-47d8-a6d1-371139ca013c.png)